registry/configuring-registry-operator.adoc

```diff
@@ -19,9 +19,9 @@ The Image Registry Operator installs a single instance of the {product-registry}
 ifndef::openshift-dedicated,openshift-rosa[]
 [NOTE]
 ====
-Storage is only automatically configured when you install an installer-provisioned infrastructure cluster on AWS, GCP, Azure, or OpenStack.
+Storage is only automatically configured when you install an installer-provisioned infrastructure cluster on IBM, AWS, GCP, Azure, or OpenStack.
```

Comment:
Could we update the platform order following alphabetical order?

Reply:
@xiuwang Done!